lms/services/digest.py

```diff
@@ -75,8 +81,14 @@ def send_instructor_email_digests(
                 # We don't have an email address for this user.
                 continue
 
+            if deduplicate:
+                task_done_key = f"instructor_email_digest::{unified_user.h_userid}::{datetime.now(timezone.utc).strftime('%Y-%m-%d')}"
```

Comment:
The key string is designed so that it should not be possible to send two instructor email digests to the same user on the same day